coderd/csp.go

```diff
@@ -14,6 +14,16 @@ type cspViolation struct {
 	Report map[string]interface{} `json:"csp-report"`
 }
 
+// @Summary Report CSP violations
+// @ID report-csp-violations
+// @Security CoderSessionToken
+// @Accept json
+// @Produce text/plain
+// @Tags General
+// @Param request body cspViolation true "Violation report"
+// @Success 200
+// @Router /csp/reports [post]
+//
```

Comment:
Nit: Same as in the previous PR, should `// logReport...` come before `@Summary`?

Reply:
Fixed